pylabnet/hardware/ni_daqs/nidaqmx_card.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file contains the pylabnet Hardware module class for a generic NI DAQ mx card.
+
+"""
+
+import nidaqmx  
+from pylabnet.utils.logging.logger import LogHandler
+
+
+class Ni_Daq_Mx_Card():
+
+    def __init__(self, device_name, logger=None):
+        """Instantiate NI DAQ mx card
+
+        :device_name: Name of NI DAQ mx card, as displayed in the measurement and automation explorer
```

Comment:
@cknaut The argument description looks cool! I was also adding expected data type 
(most probably you meant ```:device_name: (str)```) 
I would suggest to add the expected type all the time, because often it actually can be confusing whether it is supposed to be a number, a string or may be something else.

Reply:
Good idea, will implement.